docker-compose.yml

```diff
@@ -4,18 +4,25 @@ version: "3.7"
 
 services:
   php-fpm:
-    image: ghcr.io/openconext/openconext-containers/openconext-phpfpm-dev:latest
+    image: ghcr.io/openconext/openconext-basecontainers/php72-apache2-node14-composer2
```

Comment:
@quartje is this PR still valid? Since I see php72 here...

Reply:
It is not, ans seems to have been merged already. Closing.